stream-chat-android-ui-common/src/test/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModelTest.kt

```diff
@@ -138,6 +139,7 @@ internal class MessageListViewModelTest {
     }
 
     @Test
+    @Ignore("Can not be tested until we use mockk or other way to mock static function")
```

Comment:
In order to fix it we need to mock the static function. It's not possible with mockito. For that, we need to use `mockk`. If we want I can include `mockk` in this PR and fix the test